History.md

```diff
@@ -0,0 +1,23 @@
+## 0.6.1 (November 29, 2018)
+
+* [#146] Deprecate the use of the hubspot rake tasks. Deprecating these tasks
```

Comment:
I think you wanted #148 here.

Reply:
Ah, so this is a bit confusing. #146 is the PR that was merged into the `v0-stable` branch and #148 is the PR that was merged into master. Since this file is being merged into the `v0-stable` branch, I think it makes sense to link to PR #146 and then when I issue these changes against master, I'll change it to reference PR #148.

Reply:
Did you intend to reference #146 twice? for the `0.6.1` release?

Reply:
OK sounds good.